src/main/java/org/craftercms/studio/impl/v2/service/content/internal/ContentTypeServiceInternalImpl.java

```diff
@@ -196,4 +221,25 @@ protected String getContentTypePath(String contentType) {
         return normalize(contentTypeBasePathPattern.replaceFirst("\\{content-type}", contentType));
     }
 
+    protected String getContentTypePreviewImageFilename(String siteId, String contentTypeId) throws ServiceLayerException {
+        if (!siteService.exists(siteId)) {
```

Comment:
You can just use
siteService.checkSiteExists(siteId);

Reply:
Updated. Thank you!